test/distributed/_tensor/test_math_ops.py

```diff
@@ -16,29 +16,47 @@
 
 
 class DistMathOpsTest(DTensorTestBase):
-    @with_comms
-    def test_sum(self):
-        device_mesh = self.build_device_mesh()
 
+    def linear_op_reductions(self, op_str):
```

Comment:
```suggestion
    @pytest.mark.parametrize("op_str", ["all', "sum", "prod", "min", "max", "mean"])
    def test_reductions(self, op_str):
```

Reply:
For some reason the distributed tests don't work with `pytest.mark.parametrize` - that's why they all use for loops